data/benefits/openfisca/eure_et_loir_eligibilite_adefip.yml

```
@@ -0,0 +1,15 @@
+label: Aide financière d’insertion personnalisée (ADéFIP)
+institution: departement_eure_et_loir
+description: >
+  Cette aide permet aux bénéficiaires du RSA qui reprennent une activité de faire face très rapidement aux premières dépenses et aux besoins de trésorerie. Son montant varie entre 300 et 700 euros en fonction de la reprise d'activité (CDD, CDI, création d'entreprise, formation, etc.).
+conditions:
+  - Avoir un Contrat d'Engagement Réciproque (CER) ou un Parcours Personnalisé d'Accès à l'Emploi (PPAE).
+  - Ne pas avoir bénéficié de cette aide au cours des 12 derniers mois.
+  - Faire la demande avec le référent RSA <a href="https://mesdemarches.eurelien.fr/media/uploads/2023/01/05/formulaire-demande-adefip-pdf.pdf" rel="noopener" target="_blank" title="Site Mes démarches euréliens - Nouvelle fenêtre">par écrit</a> ou <a href="https://formulaires.mesdemarches.eurelien.fr/vous-etes-un-partenaire-referent-d-un-beneficiaire-rsa-1/" rel="noopener" target="_blank" title="Teleservice demande - Nouvelle fenêtre">par téléservice</a>.
```

Les usagers ne connaissent pas le mot "téléservice", c'est pour cela que le texte affiché pour le lien du téléservice est "Faire une demande en ligne"

```suggestion
  - Faire la demande avec le référent RSA <a href="https://mesdemarches.eurelien.fr/media/uploads/2023/01/05/formulaire-demande-adefip-pdf.pdf" rel="noopener" target="_blank" title="Site Mes démarches euréliens - Nouvelle fenêtre">par écrit</a> ou <a href="https://formulaires.mesdemarches.eurelien.fr/vous-etes-un-partenaire-referent-d-un-beneficiaire-rsa-1/" rel="noopener" target="_blank" title="Teleservice demande - Nouvelle fenêtre">en ligne</a>.
```